heliopy/data/helper.py

```diff
@@ -138,6 +138,10 @@ def cdf_dict(unit_string):
     ionic_charge = u.def_unit('Charged State', 1.6021766*(10**-19) * u.C)
 
     units = OrderedDict([('ratio', u.dimensionless_unscaled),
+                        ('(Unitless)', u.dimensionless_unscaled),
+                        ('(years)', u.yr),
+                        ('(days)', u.d),
+                        ('df', u.dimensionless_unscaled),
```

Comment:
Can you drop the 'df' line, I'm not sure whether something called 'df' would always be dimensionless.

Reply:
Yep! You're right, this should be a function unit rather than a universal unit.